.circleci/config.yml

```diff
@@ -684,7 +699,9 @@ jobs:
           command: |
             ./script/create_release.sh ${CIRCLE_TAG} << pipeline.parameters.KUBEAPPS_REPO >>
   local_e2e_tests:
-    machine: true
```

Comment:
It was deprecated as well, they recommend using a specific image, so I've added a param and also included it in the release process docs.